utils/countdownTimer.ts

```diff
@@ -0,0 +1,50 @@
+export type CountdownCallback = (
+    remainingSeconds: number,
+    isFinished: boolean
+) => void
+
+export const startCountdownTimer = (
+    seconds: number,
+    chosenAmountOfTimesToRepeat: number,
+    callback: CountdownCallback
+) => {
+    let timerRepeatCount = 0
+    let counterInterval:
```

Comment:
What do you think about initializing `counterInterval` as `0`? That could simplify the type and the conditional could check if it's above `0`.